napalm_junos/utils/junos_views.py

```diff
@@ -3,5 +3,11 @@
 """
 from jnpr.junos.factory import loadyaml
 from os.path import splitext
-_YAML_ = splitext(__file__)[0] + '.yml'
+from napalm_base.utils import py23_compat
+
+# Temporary fix until PYEZ provides a better solution
+if py23_compat.PY2:
```

Comment:
Yes, it raises an exception (line 121 here):

https://github.com/Juniper/py-junos-eznc/blob/master/lib/jnpr/junos/factory/factory_loader.py